src/serverlist.h

```diff
@@ -207,6 +207,8 @@ class CServerListManager : public QObject
     // count of registration retries
     int                     iSvrRegRetries;
 
+    QTextStream& tsConsoleStream = *( ( new ConsoleWriterFactory() )->get() );
```

Comment:
Shouldn't we do the initialization in the constructor? What you implemented is C++ 11, right. I am not so familiar with this construction methods. Does this has advantages over initialization in the constructor?

Reply:
Oh.  I'm not sure -- I don't mind.

Reply:
I would prefer to have it here: https://github.com/corrados/jamulus/blob/master/src/serverlist.cpp#L34
What do you think?

Reply:
OK, rebased and initialised in the constructor...

Plus moved the code _outside_ the Mutex locks, which is where I'd originally put it, before removing it for a different branch and adding it back, here in the wrong place.

I've only tested register and unregister, not the expiry.